Project.toml

```
@@ -1,7 +1,7 @@
 name = "Static"
 uuid = "aedffcd0-7271-4cad-89d0-dc628f76c6d3"
 authors = ["chriselrod", "ChrisRackauckas", "Tokazama"]
-version = "0.6.6"
+version = "0.7"
```

```suggestion
version = "0.6.7"
```

This is just to run downstream tests. Revert after seeing the results.